src/test/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPCTest.java

```diff
@@ -500,14 +550,14 @@ void testComputeTxPowerMapMultiBand() {
 			txPowerMap.get(DEVICE_C).get(UCentralConstants.BAND_2G)
 		);
 
-		// test 5G band
+		// test 5G band (only device A and device B operate in 5G)
```

Comment:
I think we should test the case that:

1. A device operates in 5G
2. A device does not operate in 5G because 5G band is not available in radio state.
3. A device does not operate in 5G due to missing of wifiscan, which was tested before my PR on the per channel TPC.

FYI, we are not supposed to rely on status to determine which band to operate in, but should rely on state. @jwoos is working on fixing it. We may have further work to distinguish and rename state vs. status.

Reply:
Ok, I'll work on this. For now, I'll keep state and status consistent in tests that I write.